packages/js/src/first-time-configuration/tailwind-components/configuration-stepper-buttons.js

```diff
@@ -4,6 +4,7 @@ import PropTypes from "prop-types";
 import { Step } from "./stepper";
 import { stepperTimingClasses } from "../stepper-helper";
 import classNames from "classnames";
+import { HIIVE_STEPS_NAMES } from "../tailwind-components/steps/constants";
```

Comment:
same as above (and even if we keep the file there, the import should be different)
```suggestion
import { HIIVE_STEPS_NAMES } from "./steps/constants";
```